releases/net462/dotnet462-changes.md

```diff
@@ -108,13 +166,46 @@
 * Allowed the debugger to determine the layout of types without an instance of the type. [211562]
 * Reduced events are being sent for telemetry to the CLR for Windows Server OSes. [211794]
 * Support resolving Windows Runtime references through simple name in .NET SDK. [219126]
+* Reliability improvements in RYUJIT complier code generation due to a missed AVX encoding case. [259784] [Added: May 2017]
+* On a machine with many processors, there might a desire to have some processes use only some processes with Server GC. For example if you have a 40-proc machine, you might want to have process 0 use only 10 procs and process 1 use only 6. So process 0 gets 10 GC heaps/threads and process 1 gets 6. Below is an example of using these Server GC configs:
+
+    ```xml
+    <configuration>
+    <runtime>
+        <gcServer enabled="true"/>
+        <GCHeapCount enabled="6"/>
+        <GCNoAffinitize enabled="true"/>
+        <GCHeapAffinitizeMask enabled="144"/>
+    </runtime>
+    </configuration>
+    ```
+
+    <GCNoAffinitize enabled="true"/> specifies to not affinitize the Server GC threads with CPUs.
+    <GCHeapCount enabled="6"/> 6 specifies you want 6 heaps. The actual # of heaps is the min of what you specify and what your proces is allowed to use.
+    <GCHeapAffinitizeMask enabled="144"/> the # is in decimal so this is 0x90, meaning you want to use those 2 bits as your process mask. The actual # of heaps is the min of what you specify, what your proces is allowed to use and the # of set bits in this mask you specify. [274126] [Added: May 2017]
```

Comment:
The XML tags here will not show in the document. They should be enclosed in code blocks.